maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenModuleProjectImporter.java

```diff
@@ -18,21 +18,65 @@
 import com.liferay.ide.core.AbstractLiferayProjectImporter;
 
 import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
 import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.maven.model.Build;
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.apache.maven.model.io.xpp3.MavenXpp3Writer;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
 import org.eclipse.core.resources.IProject;
 import org.eclipse.core.runtime.CoreException;
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.NullProgressMonitor;
 import org.eclipse.core.runtime.Status;
 import org.eclipse.m2e.core.project.IMavenProjectImportResult;
+import org.eclipse.m2e.core.project.MavenProjectInfo;
 
 /**
  * @author Andy Wu
  */
 public class MavenModuleProjectImporter extends AbstractLiferayProjectImporter
 {
+    private void addConfiguration( String name, String value, Xpp3Dom configuration )
```

Comment:
These actions of updating poms need to be initiated by the user, not during import.